prov/verbs/src/verbs_domain.c

```diff
@@ -200,6 +229,14 @@ static struct fi_ops fi_ibv_fid_ops = {
 	.ops_open = fi_no_ops_open,
 };
 
+static struct fi_ops fi_ibv_rdm_fid_ops = {
+	.size = sizeof(struct fi_ops),
+	.close = fi_ibv_domain_close,
+	.bind = fi_ibv_rdm_domain_bind,
```

Comment:
Shouldn't your patch, as it is, work for EP_MSG if the bind function is set in fi_ibv_fid_ops? If the plan is to add it in a separate patch, then it's fine. 👍